ocaml/xapi-consts/api_messages.ml

```diff
@@ -46,6 +46,8 @@ let ha_host_failed = addMessage "HA_HOST_FAILED" 3L
 
 let ha_host_was_fenced = addMessage "HA_HOST_WAS_FENCED" 4L
 
```

Comment:
Just one suggestion: this could be moved to line 138 where we have the otehr clusterting messages

Reply:
Thanks, moved.